deploy_comment.py

```diff
@@ -0,0 +1,130 @@
+import datetime
+import logging
+import os
+import sys
+
+from utils import notify_irc, parse_link_headers
+
+import requests
+
+log = logging.getLogger()
+handler = logging.StreamHandler(sys.stderr)
+log.addHandler(handler)
+log.setLevel(logging.DEBUG)
+
+logging.getLogger("requests").propagate = False
+
+GITHUB_USERNAME = os.getenv('GITHUB_USERNAME')
+GITHUB_TOKEN = os.getenv('GITHUB_TOKEN')
+
+log.info('Running deploy_comment.py: {}'.format(datetime.datetime.now()))
+repos = 'https://api.github.com/repos/'
+myself = 'addons-robot'
+
+message = ('This has been deployed to our '
+'[development server](https://addons-dev.allizom.org). Please see '
+'[push duty](https://addons.readthedocs.io/en/latest/server/push-duty.html) '
+'for when it will be deployed to production.')
```

Comment:
I believe we can calculate the "potential next deployment" with something along the lines of

```python
import datetime
import calendar
from dateutil import relativedelta

today = datetime.date.today()
specific_time = ""
next_thursday = today + relativedelta(days=1, weekday=relativedelta.TH)

if today.weekday() == calendar.THURSDAY:
    # well, this might need some more fine-tuning… We could do something like
    # if today.weekday() >= calendar.THURSDAY and do something special... or not...
    specific_time = "today"
else:
    specific_time = "on {}".format(next_thursday.isoformat())

message = "This has been deployed to our development server. The next deployment will happen {}, please see ... for more details".format(specific_time)
```

(untested, might contain syntax errors, copy with care)